data/state/peerAccount_test.go

```diff
@@ -1 +1,278 @@
+
+	err := acc.SetSchnorrPublicKey(pubKey)
+	assert.Equal(t, state.ErrNilSchnorrPublicKey, err)
+}
+
+func TestPeerAccount_SetAndGetSchnorrPublicKey(t *testing.T) {
+	t.Parallel()
+
+	acc, _ := state.NewPeerAccount(&mock.AddressMock{})
+	pubKey := []byte("SchnorrPubKey")
+
+	err := acc.SetSchnorrPublicKey(pubKey)
+	assert.Nil(t, err)
+	assert.Equal(t, pubKey, acc.GetSchnorrPublicKey())
+}
+
+func TestPeerAccount_SetRewardAddressInvalidAddress(t *testing.T) {
+	t.Parallel()
+
+	acc, _ := state.NewPeerAccount(&mock.AddressMock{})
+	err := acc.SetRewardAddress([]byte{})
+
+	assert.Equal(t, state.ErrEmptyAddress, err)
+}
+
+func TestPeerAccount_SetAndGetRewardAddress(t *testing.T) {
+	t.Parallel()
+
+	acc, _ := state.NewPeerAccount(&mock.AddressMock{})
+	addr := []byte("reward address")
+	_ = acc.SetRewardAddress(addr)
+
+	assert.Equal(t, addr, acc.GetRewardAddress())
+}
+
+func TestPeerAccount_SetInvalidStake(t *testing.T) {
+	t.Parallel()
+
+	acc, _ := state.NewPeerAccount(&mock.AddressMock{})
+
+	err := acc.SetStake(nil)
+	assert.Equal(t, state.ErrNilStake, err)
+}
+
+func TestPeerAccount_SetAndGetStake(t *testing.T) {
+	t.Parallel()
+
+	acc, _ := state.NewPeerAccount(&mock.AddressMock{})
+	stake := big.NewInt(10)
+
+	err := acc.SetStake(stake)
+	assert.Nil(t, err)
+	assert.Equal(t, stake, acc.GetStake())
+}
+
+func TestPeerAccount_SetAndGetAccumulatedFees(t *testing.T) {
+	t.Parallel()
+
+	acc, _ := state.NewPeerAccount(&mock.AddressMock{})
+	fees := big.NewInt(10)
+	acc.SetAccumulatedFees(fees)
+
+	assert.Equal(t, fees, acc.GetAccumulatedFees())
+}
+
+func TestPeerAccount_SetAndGetJailTime(t *testing.T) {
+	t.Parallel()
+
+	acc, _ := state.NewPeerAccount(&mock.AddressMock{})
+	jailTime := state.TimePeriod{}
+	acc.SetJailTime(jailTime)
+
+	assert.Equal(t, jailTime, acc.GetJailTime())
+}
+
+func TestPeerAccount_SetAndGetCurrentShardId(t *testing.T) {
+	t.Parallel()
+
+	acc, _ := state.NewPeerAccount(&mock.AddressMock{})
+	shId := uint32(5)
+	acc.SetCurrentShardId(shId)
+
+	assert.Equal(t, shId, acc.GetCurrentShardId())
+}
+
+func TestPeerAccount_SetAndGetNextShardId(t *testing.T) {
+	t.Parallel()
+
+	acc, _ := state.NewPeerAccount(&mock.AddressMock{})
+	shId := uint32(5)
+	acc.SetNextShardId(shId)
+
+	assert.Equal(t, shId, acc.GetNextShardId())
+}
+
+func TestPeerAccount_SetAndGetNodeInWaitingList(t *testing.T) {
+	t.Parallel()
+
+	acc, _ := state.NewPeerAccount(&mock.AddressMock{})
+	acc.SetNodeInWaitingList(true)
+
+	assert.True(t, acc.GetNodeInWaitingList())
+}
+
+func TestPeerAccount_SetAndGetUnStakedNonce(t *testing.T) {
+	t.Parallel()
+
+	acc, _ := state.NewPeerAccount(&mock.AddressMock{})
+	nonce := uint64(15)
+	acc.SetUnStakedNonce(nonce)
+
+	assert.Equal(t, nonce, acc.GetUnStakedNonce())
+}
+
+func TestPeerAccount_SetAndGetLeaderSuccessRate(t *testing.T) {
+	t.Parallel()
+
+	acc, _ := state.NewPeerAccount(&mock.AddressMock{})
+	increaseVal := uint32(5)
+	decreaseVal := uint32(3)
+	acc.IncreaseLeaderSuccessRate(increaseVal)
+
```

Comment:
change the empty lines in this test:  assertion right after act and an empty line between them

Reply:
Done.